src/librustdoc/html/static/js/settings.js

```diff
@@ -149,16 +147,11 @@
     function buildSettingsPage() {
         const themes = getVar("themes").split(",");
         const settings = [
-            {
-                "name": "Use system theme",
-                "js_name": "use-system-theme",
-                "default": true,
-            },
             {
                 "name": "Theme",
                 "js_name": "theme",
```

Comment:
So I suppose here’s where it would still need to be renamed to `"theme2"`.

Reply:
Indeed, thanks!